example-app/SantokuApp/src/framework/firebase/FirebaseConfig.ts

```diff
@@ -0,0 +1,45 @@
+import firebase, {ReactNativeFirebase} from '@react-native-firebase/app';
```

Comment:
型だけしか使わないから、`import type`で逃げておけばモック化しやすい感じ

```suggestion
import firebase from '@react-native-firebase/app';
import type {ReactNativeFirebase} from '@react-native-firebase/app';
```